packages/ruby-3.1/packaging

```diff
@@ -59,8 +59,8 @@ tar xzf "ruby-${RUBY_VERSION}.tar.gz"
 
 mkdir "${BOSH_INSTALL_TARGET}/bosh"
 cp gemrc "${BOSH_INSTALL_TARGET}/bosh/gemrc"
-cp compile-3.1.env "${BOSH_INSTALL_TARGET}/bosh/compile.env"
-cp runtime-3.1.env "${BOSH_INSTALL_TARGET}/bosh/runtime.env"
+envsubst '$BOSH_INSTALL_TARGET' <compile-3.1.env | tee "${BOSH_INSTALL_TARGET}/bosh/compile.env"
+envsubst '$BOSH_INSTALL_TARGET' <runtime-3.1.env | tee "${BOSH_INSTALL_TARGET}/bosh/runtime.env"
```

Comment:
This use of `envsubst` here is causing issues when the ruby package is "packaged" in an environment where `envsubst` does not exist.

In particular when running `bosh create-env` there execution context can be highly variable.

The consequence is that when `envsubst` is  not present the two files `${BOSH_INSTALL_TARGET}/bosh/compile.env` and `${BOSH_INSTALL_TARGET}/bosh/runtime.env` are empty. This results in a failure when running `bosh create-env` with an error about not being able to find the `ruby` and/or `bundle` command(s).